### ISSUE TITLE
Donated, no link for 1.16.5 only 1.15.2 or lower

### ISSUE THREAD

**Issue by PayneAS** [ImpactDevelopment/ImpactIssues#3147]
**Describe the bug**
Donated $5. Made login. Connected Minecraft account. Unable to download latest client.

**To Reproduce**
attempted to download again, more than once, still only option to install 1.15.2 or below

**Expected behavior**
I expected access to 1.16.5 as promised.

**Additional context**
Why is your discord disabled? Why haven't you made a new one? Feels like this was all a scam.

**Comment by Pearl8520** [comment]
same here. i'd like to know why

**Comment by phmage** [comment]
It's not a scam as they've been around for awhile and have one of the best utility clients around. The Discord was taken down for breaking TOS as was announced by one of the developers on Twitter and they will not be creating a new one to go around the TOS ban. 

Nightly versions are the 1.16.5 and maybe we will see a 1.17. I haven't seen if development will continue or not but we will have to make due without a Discord.

**Comment by GoingLoki** [comment]
> 
> 
> It's not a scam as they've been around for awhile and have one of the best utility clients around. The Discord was taken down for breaking TOS as was announced by one of the developers on Twitter and they will not be creating a new one to go around the TOS ban.
> 
> Nightly versions are the 1.16.5 and maybe we will see a 1.17. I haven't seen if development will continue or not but we will have to make due without a Discord.

what @phmage said, discord prevents servers that associate with any form of cheats or, in this case ("utilities"), in games.
as for viewing 1.16 verisons of impact you can create a account on the page you were redirected to and should see the nightly version on the very top of your account dashboard page after making one.

**Comment by mrmysterious33** [comment]
I payed the $5 back when the discord was still up. The computer i had before broke, so after months of going without one, i bought a new laptop. went to download the nightly installer only to find the discord down. i've been unable to get a hold of anyone since then. please help
![image](https://user-images.githubusercontent.com/56241141/121584348-9b4f5880-c9ee-11eb-8305-aa775b9d6f27.png)

**Comment by Doogie13** [comment]
Use nightly installer

**Comment by mrmysterious33** [comment]
I can't as this is a different computer and I don't have the nightly installer anymore

**Comment by Pearl8520** [comment]
> I can't as this is a different computer and I don't have the nightly installer anymore

Then install it

**Comment by PayneAS** [comment]
> > I can't as this is a different computer and I don't have the nightly installer anymore
> 
> Then install it

Where/How since the discord is no longer available?

**Comment by Pearl8520** [comment]
> > > I can't as this is a different computer and I don't have the nightly installer anymore
> > 
> > Then install it
> 
> Where/How since the discord is no longer available?

Ya do know there's a site right?

**Comment by PayneAS** [comment]
> > > > I can't as this is a different computer and I don't have the nightly installer anymore
> > > 
> > > 
> > > Then install it
> > 
> > 
> > Where/How since the discord is no longer available?
> 
> Ya do know there's a site right?

Yes, and the installer on that site only installs 1.15.2 which is what prompted this bug ticket request 
![Screenshot 2021-07-07 103817](https://user-images.githubusercontent.com/85245012/124788960-78419700-df0f-11eb-8c81-c236a13b296b.png)
If there is somewhere else I'm supposed to be installing it from, then I am clueless and have no idea since the discord is gone.

**Comment by Pearl8520** [comment]
> > > > > I can't as this is a different computer and I don't have the nightly installer anymore
> > > > 
> > > > 
> > > > Then install it
> > > 
> > > 
> > > Where/How since the discord is no longer available?
> > 
> > Ya do know there's a site right?
> 
> Yes, and the installer on that site only installs 1.15.2 which is what prompted this bug ticket request 
> ![Screenshot 2021-07-07 103817](https://user-images.githubusercontent.com/85245012/124788960-78419700-df0f-11eb-8c81-c236a13b296b.png)
> If there is somewhere else I'm supposed to be installing it from, then I am clueless and have no idea since the discord is gone.

Go to the home menu, click the 3 bars, login and then your be able to install the nightly installer

**Comment by PayneAS** [comment]
> Go to the home menu, click the 3 bars, login and then your be able to install the nightly installer

Thank you for your answer, I appreciate it. I will try it out when I get home. I hope it works!

**Comment by Pearl8520** [comment]
> 
> > Go to the home menu, click the 3 bars, login and then your be able to install the nightly installer
> 
> Thank you for your answer, I appreciate it. I will try it out when I get home. I hope it works!

Glad i could help bud
